pkgs/development/python-modules/sphinxawesome-theme/default.nix

```
@@ -0,0 +1,37 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  poetry-core,
+  sphinx,
+  beautifulsoup4
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxawesome_theme";
```

```suggestion
  pname = "sphinxawesome-theme";
```

PEP503 normalization. Use a dedicated string for the src pname.

---

got it!

---

this cause a `404` from `fetchPypi`, so i did
```nix
  pname = "sphinxawesome-theme";
  version = "5.1.4";
  pyproject = true;

  src = fetchPypi {
    inherit version;
    pname = "sphinxawesome_theme";
    hash = "sha256-OwikuKJrPo4vNaud/9JToYYJePV6Kew8izYbr/qKTtQ=";
  };

``

---

This is indeed how it's meant to be done, thank you.